gazebo_plugins/src/gazebo_ros_laser.cpp

```diff
@@ -98,6 +98,16 @@ void GazeboRosLaser::Load(sensors::SensorPtr _parent, sdf::ElementPtr _sdf)
   else
     this->topic_name_ = this->sdf->Get<std::string>("topicName");
 
+  if (!this->sdf->HasElement("timeIncrement"))
+  {
+    ROS_INFO_NAMED("gpu_laser", "GazeboRosLaser plugin missing <timeIncrement>, defaults to 0");
+    this->time_increment_ = 0;
+  }
+  else
+    this->time_increment_ = this->sdf->Get<float>("timeIncrement");
+
+  this->scan_time_ = static_cast<float>(1. / this->parent_ray_sensor_->UpdateRate());
```

Comment:
Better check if the update rate is zero, I've seen people setting it to zero on startup as a way to "turn off" the sensor.